mesheryctl/pkg/utils/platform.go

```diff
@@ -96,6 +96,25 @@ func DownloadManifests(manifestArr []Manifest, rawManifestsURL string) error {
 			}
 		}
 	}
+
+	operatorFilepath := filepath.Join(MesheryFolder, ManifestsFolder, MesheryOperator)
+	err := DownloadFile(operatorFilepath, OperatorURL)
```

Comment:
These files will be downloaded from the same release regardless of the version specified in the Meshery config file.

Reply:
@navendu-pottekkat, which is not good, correct? We should be downloading the version in accordance with what is specified in the respective `context`.

Reply:
Yes @leecalcote . We should fetch the file from the particular release specified in the context. 

@metonymic-smokey This should be a simple fix as the other files are already being downloaded in this way.